16# Resolve conflicts

abalanag opened the pull request:
Resolved conflicts between the differet versions of Bootstrap used for header and homepage
Created Thymeleaf links for all the CSS files.

abalanag committed:
16# Resolve conflicts

Resolved conflicts between the differet versions of Bootstrap used for header and homepage
Created Thymeleaf links for all the CSS files.

danutchindris commented on src/main/resources/templates/header.html:
```diff
@@ -1,98 +1,105 @@
 <!doctype html>
 
-<html xmlns:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" th:fragment="header-info">
```
I don't think this is a good idea. Are you going to inject the `<html>` element inside another full HTML document? Doesn't sound right. You should inject the `<header>` element or something like that.

abalanag replied:
Done :)

danutchindris reviewed (changes requested):
Please see the review comments.

abalanag committed:
16# Updated project

Moved the thymeleaf fragment from HTML as was requested on the pull request comment

danutchindris reviewed (approved):
Looks OK.
